### DISCUSSION TITLE
Update individual tools and honeypots?

### DISCUSSION THREAD

**Discussion by TonAtLacework** [telekom-security/tpotce#981] [Q&A]
Hi all! First post here.
I understand that T-Pot gets updated every, say, 6 months. However, the tools that make up T-Pot update much more frequent. Is there an approved method on how to update the individual tools?

For example: I have issues with Spiderfoot 3.4.0 which is part of T-Pot. Before asking on their forums i want to update to the latest version which is 3.5. What is the approved way to update Spiderfoot (or whatever else) to the latest version?

**Comment by t3chn0m4g3** [comment]
All the tools and honeypots are updated on a loosely basis between releases.

**Reply by TonAtLacework** [reply]
Thanks! Any ETA on Spiderfoot 3.5?
